java/code/src/com/redhat/rhn/frontend/xmlrpc/proxy/ProxyHandler.java

```diff
@@ -364,6 +379,224 @@ public byte[] containerConfig(User loggedInUser, String proxyName, Integer proxy
         }
     }
 
+
+    /**
+     * Deploy a proxy container on given salt minion. It expects that the images are installed as RPMs.
+     *
+     * @param loggedInUser the current user
+     * @param hostId the ID of the target minion
+     * @param parentFqdn the FQDN of the server the proxy uses
+     * @param proxyPort the SSH port the proxy listens on
+     * @param maxCache the maximum memory cache size
+     * @param email the email of proxy admin
+     * @param rootCA CA certificate in PEM format
+     * @param intermediateCAs a list of intermediate CAs in PEM format
+     * @param proxyCert proxy certificate in PEM format
+     * @param proxyKey proxy private key in PEM format
+     *
+     * @return 1 on success
+     *
+     * @apidoc.doc Deploy a proxy container on given salt minion. It expects that the images are installed as RPMs.
+     * @apidoc.param #session_key()
+     * @apidoc.param #param("int", "hostId", "The ID of the target minion")
```

Comment:
3 parameter version of this must be `#param_desc` (applies to all below)
```suggestion
     * @apidoc.param #param_desc("int", "hostId", "The ID of the target minion")
```

Reply:
This was broken everywhere in that file. Fixed now.